libobs/obs.c

```diff
@@ -1420,6 +1428,14 @@ void obs_shutdown(void)
 		bfree(obs->safe_modules.array[i]);
 	da_free(obs->safe_modules);
 
+	for (size_t i = 0; i < obs->disabled_modules.num; i++)
+		bfree(obs->disabled_modules.array[i]);
+	da_free(obs->disabled_modules);
```

Comment:
Add a blank line after the `if` contents, or enclose the `if` contents.
```suggestion
	for (size_t i = 0; i < obs->disabled_modules.num; i++)
		bfree(obs->disabled_modules.array[i]);

	da_free(obs->disabled_modules);
```

```suggestion
	for (size_t i = 0; i < obs->disabled_modules.num; i++) {
		bfree(obs->disabled_modules.array[i]);
	}
	da_free(obs->disabled_modules);
```